XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -149,7 +149,10 @@ private void Build()
                     ShowTab(VCpuMemoryEditPage = new CPUMemoryEditPage());
                     ShowTab(StartupOptionsEditPage = new BootOptionsEditPage());
                     ShowTab(VMHAEditPage = new VMHAEditPage {VerticalTabs = verticalTabs});
-                    ShowTab(usbEditPage = new USBEditPage());
+                    if (!Helpers.FeatureForbidden(xenObjectCopy, Host.RestrictUsbPassthrough))

```

Comment:
The USB page should be placed between the GPU and Advanced options pages.
The same for the main window: the USB tab should be placed between GPU and Console

Reply:
Good idea.